pulpcore/pulpcore/app/serializers/repository.py

```diff
@@ -249,6 +249,9 @@ class Meta:
             'name', 'base_path', 'publisher', 'publication', 'base_url', 'repository',
```

Comment:
It's a bit strange that we have ```base_path``` mapped to ```base_url``` instead of using one name consistently (not part of this PR specifically)

Reply:
I'm not sure I understand. They're two separate things: base_path is like `foo/bar` and base_url is the full url like `localhost:8000/foo/bar`.

Reply:
I was referring to:

```
 base_url = BaseURLField(
        source='base_path', read_only=True,
        help_text=_('The URL for accessing the publication as defined by this distribution.')
)
```

But I think maybe I'm just not understanding the code semantics.